apps/sms/views/shared/test/unit/dialog_test.js

```diff
@@ -5,6 +5,7 @@
 */
 'use strict';
 
+require('/shared/js/event_safety.js');
```

Comment:
I'm not comfortable adding something that's not a mock here, as it will bring timeouts and thus unpredictability into the test.

Ideally I'd like mocking event_safety (either with sinon only, or with an empty mock + sinon (preferred)) and that in the test we test both cases (with/without the event parameter) unless you change the event_safety lib.